src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -383,6 +383,42 @@ inline std::istream& operator>>(std::istream& is, SchedulingCoreType& core_type)
 }
 /** @endcond */
 
+enum class MaxThreadsPerStream {
+    AUTO,          //!<  Using all threads per platform for one stream. Will create sub stream on dual socket platform.
+    PER_PLATFORM,  //!<  Using all threads per platform for one stream even on dual socket platform.
+    PER_SOCKET,    //!<  Using all threads per socket for one stream on dual socket platform.
+};
```

Comment:
No. enable_cpu_pinning and max_threads_per_stream are two independent properties.

Reply:
The question is still here
> E.g. pinning=False # of sockets > 1, MaxThreads = PER_SOCKET
How with it work when pinning is off (so threads can migrate from socket to socket), but PER_SOCKET is enabled?

Reply:
When PER_SOCKET is enabled, CPU plugin create one stream with the number of threads on one socket for latency mode. 

Since cpu pinning is false, StreamsExecutor will create arena on this socket without pinning and let TBB schedule all threads on this socket. TBB will not migrate threads of this arena to another socket.